src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_Diagnostics.cs

```diff
@@ -25,8 +25,16 @@ internal partial class CodeAnalysisService
         {
             using (RoslynLogger.LogBlock(FunctionId.CodeAnalysisService_CalculateDiagnosticsAsync, arguments.ProjectId.DebugName, CancellationToken))
             {
+                IDisposable booster = null;
+
                 try
                 {
+                    if (arguments.ForcedAnalysis)
```

Comment:
this won't make "Fix All" any faster. but it will make its perf to be more stable since it runs as same priority as other processes.